clients/vulnerable-code/src/main/kotlin/VulnerableCodeService.kt

```diff
@@ -39,7 +39,7 @@ import retrofit2.http.Path
  */
 interface VulnerableCodeService {
     companion object {
-        val JSON_MAPPER = JsonMapper().setPropertyNamingStrategy(PropertyNamingStrategies.SNAKE_CASE)
+        private val JSON_MAPPER = JsonMapper().setPropertyNamingStrategy(PropertyNamingStrategies.SNAKE_CASE)
```

Comment:
By convention, I'd like to keep this public as part of the client API, so users of this generic library are free to use the same mapper as the client implementation. We just had a [concrete use-case](https://github.com/oss-review-toolkit/ort/pull/3546/files#diff-c5a33a246601c5ca60c4e3f4b23451e8ba0370b757b2122cd3390245727491b2R137) for this in the FossId implementation, so it might happen here, too.

Reply:
I've dropped that commit.